extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/tunnel/RestrictedGuacamoleTunnelService.java

```diff
@@ -180,8 +187,24 @@ protected ModeledConnection acquire(RemoteAuthenticatedUser user,
             @Override
             public int compare(ModeledConnection a, ModeledConnection b) {
 
-                return getActiveConnections(a).size()
-                     - getActiveConnections(b).size();
+                // Active connections
+                int Ca = getActiveConnections(a).size();
```

Comment:
Any reason the variables here don't follow the `headlessCamelCase` convention used by the other Java code?

Depending on why you went this route, I'm not sure this needs to be changed, but this function is slightly hard to read with how similar all the variables look.

Anyway, not necessarily an issue worth further iterations, but I'm interested to hear what you were thinking at the time and what your thoughts are now.